Add DS to dashboard page (#240)
seqeralabs/docs: platform_versioned_docs/version-24.1/monitoring/dashboard.mdx

### Export data

Select **Export data** in the filter panel near the top of the page to export dashboard data, based on the filters you have applied, in a CSV file.

### Pipeline runs per organization

The pipeline run totals for your selected filters are displayed for each organization that you have access to.

Depending on the filter selected, each card details a separate workspace or organization. Total pipeline runs for each organization are arranged by workspace and status.

For a detailed view, you can do one of the following:

- Select a pipeline run integer value in the table to navigate to a pipeline run list filtered by the status and time range selected.
- Select a workspace name in the table to navigate to a pipeline run list filtered by the workspace selected.

## Data Studios

You can explore the status of data studio sessions in your organizational workspaces. On the **Dashboard** page, select **Data Studios**. The following statuses are listed with the number of data studio sessions in each status:

> - `ning`

Reviewer: Regex error match on `Run`? Needs to be fixed to `Running`

> - `ning`

Reviewer 2: ```suggestion
- `Running`
```

tags: [dashboard, runs, monitoring]
---

> The Seqera Platform **Dashboard** is accessed from the user menu and provides an overview of pipeline runs and data studio sessions in your personal and organization workspaces.

Reviewer: Data Studios is not yet supported in personal workspaces. Let's be explicit:

```suggestion
The Seqera Platform **Dashboard** is accessed from the user menu and provides an overview of:
- Pipeline Runs in your personal and organization workspaces
- Data Studio sessions in your organization workspaces only
```

tags: [dashboard, runs, monitoring]
---

> The Seqera Platform **Dashboard**, accessed from the user menu, provides an overview of runs and data studios sessions in your personal and organization workspaces.

Reviewer: ```suggestion
The Seqera Platform **Dashboard**, accessed from the user menu, provides an overview of pipeline runs and data studios sessions in your personal and organization workspaces.
```

> ## Runs

Reviewer: ```suggestion
## Pipeline Runs
```

> You can explore the status of runs in your personal and in organizational workspaces. On the **Dashboard** page page, select **Runs**.

Reviewer: ```suggestion
You can explore the status of pipeline runs in your personal and in organizational workspaces. On the **Dashboard** page, select **Pipeline Runs**.
```

For a detailed view of runs, you can do one of the following:

- Select a pipeline run integer value in the table to navigate to a pipeline run list filtered by the status and time range selected.
- Select a workspace name in the table to navigate to a pipeline run list filtered by the workspace selected.

## Data Studios

You can explore the status of data studio sessions in your organizational workspaces. On the **Dashboard** page, select **Data Studios**. The following statuses are listed with the number of data studio sessions in each status:

- `Running`
- `Errored`
- `Starting`
- `Stopping`
- `Stopped`

### Filters and summary

The **Dashboard** view defaults to all organizations and workspaces you can access. Select the **View** dropdown menu to filter by organizations and workspaces. Select a status in the table to navigate to a list filtered by the status selected.

> ## Export data

Reviewer: I think this needs to be indented to 3 times again because there's another `Export data` entry above (which I missed first time around)
```suggestion
### Export data
```

For a detailed view of runs, you can do one of the following:

- Select a pipeline run integer value in the table to navigate to a pipeline run list filtered by the status and time range selected.
- Select a workspace name in the table to navigate to a pipeline run list filtered by the workspace selected.

## Data Studios

You can explore the status of data studio sessions in your organizational workspaces. On the **Dashboard** page, select **Data Studios**.

A summary of the number of data studio sessions in each status is available:

- `Running`
- `Errored`
- `Starting`
- `Stopping`
- `Stopped`

### Filters and summary

> The **Dashboard** view defaults to all organizations and workspaces you can access. Select the **View** dropdown menu to filter by organization(s) and workspace(s).

Author: @justinegeffen, I've never quite understood _(s)_. It is a bit self evident that you can filter by one or more, so it reads awkwardly whenever I see this construction in any documentation. It feels overly pedantic.

Reviewer 2: I agree - and it was also inconsistent with the rest of the references. Thanks for catching it.

> - Select a run integer value in the table to navigate to a run list filtered by the status and time range selected.

Reviewer: ```suggestion
- Select a pipeline run integer value in the table to navigate to a pipeline run list filtered by the status and time range selected.
```

> - Select a workspace name in the table to navigate to a run list filtered by the workspace selected.

Reviewer: ```suggestion
- Select a workspace name in the table to navigate to a pipeline run list filtered by the workspace selected.
```

## Data Studios

> If your instance of Seqera Platform has [Data Studios][ds] enabled, you can explore the status of data studios in your organizational workspaces. On the **Dashboard** page page, select **Data Studios**.

Reviewer: Data Studios is enabled by default.

```suggestion
You can explore the status of data studios in your organizational workspaces. On the **Dashboard** page, select **Data Studios**.
```

> A summary of the number of data studios in each state is available:

Reviewer: ```suggestion
A summary of the number of data studio sessions in each status is available:
```

- `Running`
- `Errored`
- `Starting`
- `Stopping`
- `Stopped`

### Filters and summary

> The **Dashboard** view defaults to all organizations and workspaces you can access. Select the **View** dropdown menu to filter by data studios session status.

Reviewer: ```suggestion
The **Dashboard** view defaults to all organizations and workspaces you can access. Select the **View** dropdown menu to filter by organization(s) and workspace(s).
```

> You can filter by data studios session state. To filter the set of runs, select **Filter**. When a filter is applied, the button icon and color changes.

Reviewer: ```suggestion
- Select a status in the table to navigate to a Data Studios session list filtered by the status selected.
```

> ### Export data

Reviewer: Exporting is for both Pipeline Runs and Data Studios.

```suggestion
## Export data
```

> Select **Export data** in the filter panel near the top of the page to export dashboard data for the organization and workspace in a CSV file.

Reviewer: ```suggestion
Select **Export data** in the view panel near the top of the page to export dashboard data for the organization(s) and workspace(s) selected in a CSV file.
```
